multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh

```diff
@@ -1,6 +1,6 @@
 <#assign words = ["abstract", "assert", "boolean", "break", "byte", "case", "catch", "char", "class", "const", "continue", "default", "double", "do", "else", "enum", "extends", "false", "final", "finally", "float", "for", "goto", "if", "implements", "import", "instanceof", "int", "interface", "long", "native", "new", "null", "package", "private", "protected", "public", "return", "short", "static", "strictfp", "super", "switch", "synchronized", "this", "throw", "throws", "transient", "true", "try", "void", "volatile", "while"]>
 <#function calculateSafeName propertyName ender>
-    <#if words?seq_contains(propertyName?uncap_first)>
+    <#if words?seq_contains(propertyName?uncap_first) || propertyName?matches("[0-9].*")>
```

Comment:
I love it. 
Same fix should be done for Openapi files.
Also we use semver to change product versión number. So if you can bump lower version number and also update the Readme.md I think we can approve the merge.

Reply:
Done

Reply:
I think I’ve fixed the issues for enums, there was more templates parts to update.

Reply:
It should fix openapi in the same time, I runned the tests to validate I don’t break existing code.